src/crypto/index.js

```diff
@@ -288,11 +295,19 @@ function _maybeUploadOneTimeKeys(crypto) {
 
     crypto._oneTimeKeyCheckInProgress = true;
     q().then(() => {
+        if (currentKeyCount !== undefined && isFinite(currentKeyCount)) {
+            // previously this parameter was not defined.
```

Comment:
This was done by intention. As the value would be `undefined` I can use the call in _onSyncCompleted without worrying that I break up with the old behaviour.